vite.config.ts

```diff
@@ -3,13 +3,17 @@ import { NodeModulesPolyfillPlugin } from '@esbuild-plugins/node-modules-polyfil
 import react from '@vitejs/plugin-react-swc'
 import rollupNodePolyFill from 'rollup-plugin-polyfill-node'
 import { defineConfig, loadEnv } from 'vite'
+import { nodePolyfills } from 'vite-plugin-node-polyfills'
 
 // https://vitejs.dev/config/
 export default defineConfig(({ command, mode }) => {
   const envVariables = loadEnv(mode, process.cwd())
   return {
-    plugins: [react()],
-    // Required because the CatalystClient tries to access it
+    plugins: [
+      react(),
+      // The catalyst client requires node polyfills to work in the browser.
+      nodePolyfills()
```

Comment:
Have you tried removing the other polyfills to see if it works without them? I'm not sure what would happen if we don't remove them, maybe we'll have the code polyfilled twice.

Reply:
Seems to work :D